analysis_options.yaml

```diff
@@ -0,0 +1,15 @@
+analyzer:
+  strong-mode: true
+# Upgrade lints to errors that will break Google3
```

Comment:
Why do these break Google3 if they aren't errors externally? They don't affect downstream packages in any way.